src/Components/Common/Export.tsx

```diff
@@ -0,0 +1,98 @@
+import DropdownMenu, {
+  DropdownItem,
+  DropdownItemProps,
+} from "../../Components/Common/components/Menu";
+import CareIcon from "../../CAREUI/icons/CareIcon";
+import ButtonV2 from "../../Components/Common/components/ButtonV2";
+import useExport from "../../Common/hooks/useExport";
+
+interface ExportItem {
+  dropdownOptions?: DropdownItemProps;
+  exportType?: "csv" | "json";
+  exportFilePrefix?: string;
+  exportLabel: string;
+  exportFunction: any;
+}
```

Comment:
@khavinshankar how about simplifying the props since we already are passing it inside Export button so it's contextually understood following props are for export...

```suggestion
interface ExportItem {
  options?: DropdownItemProps;
  type?: "csv" | "json";
  filePrefix?: string;
  label: string;
  action / callback: any;
}
```

Reply:
done